definitions/ext-datadog_mysql/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: DATADOG_MYSQL
```

Comment:
We might want to make this a more generic EXT-MYSQL.  Let me check with product management on our side.

Reply:
Ok I have confirmed with @jfjoly that we should make this more generic, can you change the type to `MYSQL`?

Also, you'll have to update the directory to be `ext-mysql`.

Thanks!

Reply:
Switched over to MYSQL and `ext-mysql` for the directory!